<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CppTools::ClazyChecks</class>
 <widget class="QWidget" name="CppTools::ClazyChecks">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>220</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Each level adds checks to the previous level. For more information, see &lt;a href=&quot;https://github.com/KDE/clazy&quot;&gt;clazy's homepage&lt;/a&gt;.</string>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="clazyRadioDisabled">
     <property name="text">
      <string>Disabled</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="clazyRadioLevel0">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Level 0: No false positives</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="clazyRadioLevel1">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Level 1: Very few false positives</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="clazyRadioLevel2">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Level 2: More false positives</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="clazyRadioLevel3">
     <property name="toolTip">
      <string>Not always correct, possibly very noisy, might require a knowledgeable developer to review, might have a very big rate of false-positives, might have bugs.</string>
     </property>
     <property name="text">
      <string>Level 3: Experimental checks</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>34</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
